R/reportEDGET.R

```diff
@@ -327,9 +327,12 @@ toolReportEDGET <- function(output_folder = ".",
   reportStockAndSales <- function(annual_mileage, load_factor) {
     year_c <- construction_year <- Stock <- Sales <- vintage_demand_vkm <- fct <- category <- NULL
     vint <- vintageReport(load_factor)
-    vint[, stock_demand := sum(vintage_demand_vkm), by=c("year", "region", "vehicle_type", "technology")]
-    vint[, sales_demand := full_demand_vkm - stock_demand]
-    vint <- unique(vint[, c("construction_year", "vintage_demand_vkm", "full_demand_vkm") := NULL])
+    vint[, tot_vint_demand := sum(vintage_demand_vkm), by=c("year", "region", "vehicle_type", "technology")]
```

Comment:
What gets summed up here?

Reply:
the vintages from each year.

Reply:
Are there different vintages for the same technology and vehicle time for each year? Or what am I missing?